src/test/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverageIntegrationTest.java

```diff
@@ -0,0 +1,259 @@
+package org.broadinstitute.hellbender.tools.walkers.coverage;
+
+import com.opencsv.CSVReader;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class DepthOfCoverageIntegrationTest extends CommandLineProgramTest {
+
+    private String getDoCExtensionFromFile(File docOutputFile, String basename) {
+        String[] split = docOutputFile.getAbsolutePath().split(basename);
+        if (split.length == 2) {
+            return split[1];
+        } else if (split.length == 1) {
+            return "";
```

Comment:
When would this happen? Do you expect there to be files with no extensions generated by the tool?

Reply:
Yes, the root file is the locus summary output.